wordpress-mobile/WordPress-Android
# Issue/7137 update sites empty

### Fix
Update ***Sites*** tab empty state as described in https://github.com/wordpress-mobile/WordPress-Android/issues/7137.

### Test
0. Create account with no sites.
1. Go to ***Sites*** tab.
2. Notice updated layout.
